### DISCUSSION TITLE
Actividad 5

### DISCUSSION THREAD

**Discussion by anvaldes** [JavierBZ/Tarea3-IIC3697#19] [Q&A] [answered]
Tengo una duda: Cuandos nos hablan del DQN normal es simplemente la misma red pero sin la capa las capas fully conected de adv, cierto?

**Comment by JavierBZ** [comment, marked as answer]
No, es la red que vieron en clases. Sin la capa adv estaria estimando V(s), la red DQN original estima Q(s,a).

**Reply by anvaldes** [reply]
Buena! Gracias!!

**Reply by valeria-adasme** [reply]
Para los parámetros entrenables bastaría con quitarle los parámetros que añaden las ramas de V(s) y A(s,a)??

**Reply by JavierBZ** [reply]
No, con eso solo te quedas con la parte convolucional. Tienen que crear la red original como la del ejemplo en clases (de flappy bird)
